src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -28,7 +28,7 @@ public struct MetricPoint
 
         private readonly AggregationType aggType;
 
-        private HistogramBuckets histogramBuckets;
+        private MetricPointOptionalComponents mpComponents;
```

Comment:
nit: Rename this to `optionalComponents`?

Reply:
will do as soon as I get get this back to main branch. (working on it now)